### PR TITLE
ROX-26563: Document policy CRD fields

### PR DESCRIPTION
**Pull request by kylape**
This will help with kubectl explain for users that want to build a policy outside the Central UI.

I also set some fields as required to better reflect required policy fields in the Central API.

Reviewers, please feel free to push changes to this branch to fix any gaps or inaccuracies in my changes.  This was a quick-n-dirty pass at documentation to get `kubectl explain` go a demo-able state.

### CODE REVIEW

File: config-controller/api/v1alpha1/policy_types.go [hunk 1]
```diff
@@ -38,27 +38,39 @@ type EnforcementAction string
 type SecurityPolicySpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^[^\n\r\$]{5,128}$`
-	PolicyName string `json:"policyName,omitempty"`
+	// PolicyName is the name of the policy as it appears in Central.  Note that changing this value will rename the policy in Central.  This field must be unique.
```
Comment (clickboo) [inline config-controller/api/v1alpha1/policy_types.go:41]: We should just replicate the swagger doc here to explain the field. And then add more stuff to explain any more policy as code specific stuff.

**Comment by clickboo** [comment]
+ @kcarmichael08

File: config-controller/api/v1alpha1/policy_types.go (hunk 1 above)
Comment (kylape) [reply config-controller/api/v1alpha1/policy_types.go:41]: Do you mean just grab the doc from swagger and copy over by hand?  Or did you mean something more fancy like a script to keep them in sync?

**Comment by kylape** [comment]
Made another pass on the doc.  Also copied doc strings over to protobuf.

File: config-controller/api/v1alpha1/policy_types.go [hunk 2]
```diff
@@ -38,31 +38,47 @@ type EnforcementAction string
...
```
Comment (vjwilson) [inline config-controller/api/v1alpha1/policy_types.go:41]: nit: "... as it appears in Central" is a little vague. Maybe we should hash out a standard way to refer to things like this.

Not worth changing just for this, but here's a brainstorm we could consider in the future:
```suggestion
	// PolicyName is the name of the policy as it appears in the API and UI.  Note that changing this value will rename the policy as stored in the database.  This field must be unique.
```

File: config-controller/api/v1alpha1/policy_types.go [hunk 3]
```diff
@@ -38,31 +38,47 @@ type EnforcementAction string
...
+	Scope []Scope `json:"scope,omitempty"`
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Enum=UNSET_SEVERITY;LOW_SEVERITY;MEDIUM_SEVERITY;HIGH_SEVERITY;CRITICAL_SEVERITY
-	Severity           string              `json:"severity,omitempty"`
+	// Severity defines how severe a violation from this policy is.  Possible values are UNSET_SEVERITY, LOW_SEVERITY, MEDIUM_SEVERITY, HIGH_SEVERITY, and CRITICAL_SEVERITY.
+	Severity string `json:"severity"`
+	// Enforcement lists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT.
 	EnforcementActions []EnforcementAction `json:"enforcementActions,omitempty"`
-	Notifiers          []string            `json:"notifiers,omitempty"`
+	// Notifiers is a list of notifier IDs that should be triggered when a violation from this policy is identified.  IDs should be in the form of a UUID and are found in the Central API.
```
Comment (vjwilson) [inline config-controller/api/v1alpha1/policy_types.go:70]: nit: phrasing

```suggestion
	// Notifiers is a list of IDs of the notifiers that should be triggered when a violation from this policy is identified.  IDs should be in the form of a UUID and are found through the Central API.
```

File: config-controller/api/v1alpha1/policy_types.go [hunk 4]
```diff
@@ -38,31 +38,47 @@ type EnforcementAction string
...
 	MitreAttackVectors []MitreAttackVectors `json:"mitreAttackVectors,omitempty"`
-	CriteriaLocked     bool                 `json:"criteriaLocked,omitempty"`
-	MitreVectorsLocked bool                 `json:"mitreVectorsLocked,omitempty"`
-	IsDefault          bool                 `json:"isDefault,omitempty"`
+	// Read-only field. If true, the policy's criteria fields are rendered read-only.
+	CriteriaLocked bool `json:"criteriaLocked,omitempty"`
+	// Read-only field. If true, the policy's MITRE ATT&CK fields are rendered read-only.
+	MitreVectorsLocked bool `json:"mitreVectorsLocked,omitempty"`
+	// Read-only field. Indicates the policy is a default policy if true and a custom policy if false.
+	IsDefault bool `json:"isDefault,omitempty"`
```
Comment (vjwilson) [inline config-controller/api/v1alpha1/policy_types.go:81]: @kylape @clickboo @c-du 

TODO:  come up with a plan for the future of default policies. It's weird to allow a customer to designate a bespoke, custom policy a "default" when we use the term to mean "comes out-of-the-box when you install ACS".

**Review by vjwilson — APPROVED** [review]
Feel free to resolve all my discussions; I just wanted to point up some places we could clean up the language later.

File: config-controller/api/v1alpha1/policy_types.go (hunk 4 above)
Comment (kylape) [reply config-controller/api/v1alpha1/policy_types.go:81]: https://issues.redhat.com/browse/ROX-26757

File: generated/api/v1/alert_service.swagger.json [hunk 5]
```diff
@@ -1381,64 +1381,75 @@
...
-          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates."
+          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates.\nLists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT."
```
Comment (kcarmichael08) [inline generated/api/v1/alert_service.swagger.json:1445]: see previous comment on this string

File: generated/api/v1/detection_service.swagger.json [hunk 6]
```diff
@@ -1075,64 +1075,75 @@
...
-          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates."
+          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates.\nLists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT."
```
Comment (kcarmichael08) [inline generated/api/v1/detection_service.swagger.json:1139]: see previous comment on this string

File: generated/api/v1/policy_service.swagger.json [hunk 7]
```diff
@@ -649,64 +649,75 @@
...
-          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates."
+          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates.\nLists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT."
```
Comment (kcarmichael08) [inline generated/api/v1/policy_service.swagger.json:713]: "Possible value are" - should be "possible values are" ?

extra period after "and" in last sentence

File: image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml [hunk 8]
```diff
@@ -42,18 +42,30 @@ spec:
...
+                description: Description is a free-form text description of this policy.
                 pattern: ^[^\$]{0,800}$
                 type: string
               disabled:
+                description: Disabled toggles whether or not this policy will be executing
+                  and actively firing alerts.
                 type: boolean
               enforcementActions:
+                description: Enforcement lists the enforcement actions to take when
+                  a violation from this policy is identified.  Possible value are
```
Comment (kcarmichael08) [inline image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml:65]: see previous comment on this string

File: proto/storage/policy.proto [hunk 9]
```diff
@@ -16,26 +16,38 @@ option java_package = "io.stackrox.proto.storage";
...
   repeated Exclusion exclusions = 21;
+  // Defines clusters, namespaces, and deployments that should be included in this policy.  No scopes defined includes everything.
   repeated Scope scope = 11;
 
+  // Defines how severe a violation from this policy is.  Possible values are UNSET_SEVERITY, LOW_SEVERITY, MEDIUM_SEVERITY, HIGH_SEVERITY, and CRITICAL_SEVERITY.
   Severity severity = 12; // @gotags: search:"Severity,store"
   // FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.
   // FAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.
   // FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates.
+  // Lists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT.
```
Comment (kcarmichael08) [inline proto/storage/policy.proto:48]: see previous comments on this string (values, extra period)

File: config-controller/api/v1alpha1/policy_types.go [hunk 10]
```diff
@@ -38,31 +38,47 @@ type EnforcementAction string
...
+	// Exclusions define deployments or images that should be excluded from this policy.
+	Exclusions []Exclusion `json:"exclusions,omitempty"`
+	// Scope defines clusters, namespaces, and deployments that should be included in this policy.  No scopes defined includes everything.
+	Scope []Scope `json:"scope,omitempty"`
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Enum=UNSET_SEVERITY;LOW_SEVERITY;MEDIUM_SEVERITY;HIGH_SEVERITY;CRITICAL_SEVERITY
-	Severity           string              `json:"severity,omitempty"`
+	// Severity defines how severe a violation from this policy is.  Possible values are UNSET_SEVERITY, LOW_SEVERITY, MEDIUM_SEVERITY, HIGH_SEVERITY, and CRITICAL_SEVERITY.
+	Severity string `json:"severity"`
+	// Enforcement lists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT.
```
Comment (kcarmichael08) [inline config-controller/api/v1alpha1/policy_types.go:68]: see previous comment on this string (values, extra period)

File: config-controller/config/crd/bases/config.stackrox.io_securitypolicies.yaml [hunk 11]
```diff
@@ -42,18 +42,30 @@ spec:
...
+                description: Description is a free-form text description of this policy.
                 pattern: ^[^\$]{0,800}$
                 type: string
               disabled:
+                description: Disabled toggles whether or not this policy will be executing
+                  and actively firing alerts.
                 type: boolean
               enforcementActions:
+                description: Enforcement lists the enforcement actions to take when
+                  a violation from this policy is identified.  Possible value are
```
Comment (kcarmichael08) [inline config-controller/config/crd/bases/config.stackrox.io_securitypolicies.yaml:65]: see previous comment on this string (values, extra period)

File: generated/api/v1/policy_service.swagger.json [hunk 12]
```diff
@@ -975,64 +987,75 @@
...
```
Comment (kcarmichael08) [inline generated/api/v1/policy_service.swagger.json:1051]: see previous comment on this string

File: proto/storage/policy.proto [hunk 13]
```diff
@@ -79,13 +92,18 @@ enum EventSource {
...
+  // The set of policies groups that make up this section.  Each group can be considered an individual criterion.
   repeated PolicyGroup policy_groups = 3;
 }
 
 message PolicyGroup {
+  // Defines which field on a deployment or image this PolicyGroup evaluates.  See https://docs.openshift.com/acs/operating/manage-security-policies.html#policy-criteria_manage-security-policies for a complete list of possible values.
   string field_name = 1;
+  // Determines if the values are combined with an OR or an AND.  Defaults to OR.
   BooleanOperator boolean_operator = 2;
+  // Determines if the evaluation of this PolicyGroup is negated.  Default to false.
```
Comment (kcarmichael08) [inline proto/storage/policy.proto:104]: default - should be "defaults" ?

File: proto/storage/policy.proto [hunk 14]
```diff
@@ -79,13 +92,18 @@ enum EventSource {
 
 message PolicySection {
   string section_name = 1;
+  // The set of policies groups that make up this section.  Each group can be considered an individual criterion.
```
Comment (kcarmichael08) [inline proto/storage/policy.proto:95]: policies - should be policy ?

File: image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml [hunk 15]
```diff
@@ -129,48 +149,82 @@ spec:
...
                 type: array
               rationale:
                 type: string
               remediation:
+                description: Remediation describes how to remediate a violation of
+                  this policy.
                 type: string
               scope:
+                description: Scope defines clusters, namespaces, and deployments that
+                  should be included in this policy.  No scopes defined includes everything.
```
Comment (kcarmichael08) [inline image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml:227]: see previous comment with suggestion for last sentence wording.

File: image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml [hunk 16]
```diff
@@ -129,48 +149,82 @@ spec:
...
+                            description: Negate determines if the evaluation of this
+                              PolicyGroup is negated.  Default to false.
                             type: boolean
                           values:
+                            description: Values is the list of values for the specified
+                              field
                             items:
                               properties:
                                 value:
+                                  description: Value is simply the string value
```
Comment (kcarmichael08) [inline image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml:202]: period after value?

File: image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml [hunk 17]
```diff
@@ -129,48 +149,82 @@ spec:
...
+                              or image this PolicyGroup evaluates.  See https://docs.openshift.com/acs/operating/manage-security-policies.html#policy-criteria_manage-security-policies
+                              for a complete list of possible values.
                             type: string
                           negate:
+                            description: Negate determines if the evaluation of this
+                              PolicyGroup is negated.  Default to false.
                             type: boolean
                           values:
+                            description: Values is the list of values for the specified
+                              field
```
Comment (kcarmichael08) [inline image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml:198]: period after field?

File: image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml [hunk 18]
```diff
@@ -67,12 +79,16 @@ spec:
                   type: string
                 type: array
               eventSource:
+                description: EventSource describes which events should trigger execution
+                  of this policy
```
Comment (kcarmichael08) [inline image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml:83]: period after policy?

File: generated/storage/policy.pb.go [hunk 19]
```diff
@@ -394,31 +394,44 @@ type Policy struct {
...
+	// Free-form text description of this policy.
+	Description string `protobuf:"bytes,3,opt,name=description,proto3" json:"description,omitempty" search:"Description"` // @gotags: search:"Description"
+	Rationale   string `protobuf:"bytes,4,opt,name=rationale,proto3" json:"rationale,omitempty"`
+	// Describes how to remediate a violation of this policy.
+	Remediation string `protobuf:"bytes,5,opt,name=remediation,proto3" json:"remediation,omitempty"`
+	// Toggles whether or not this policy will be executing and actively firing alerts.
+	Disabled bool `protobuf:"varint,6,opt,name=disabled,proto3" json:"disabled,omitempty" search:"Disabled"` // @gotags: search:"Disabled"
+	// List of categories that this policy falls under.  Category names must already exist in Central.
+	Categories []string `protobuf:"bytes,7,rep,name=categories,proto3" json:"categories,omitempty" search:"Category,store"` // @gotags: search:"Category,store"
+	// Describes which policy lifecylce stages this policy applies to.  Choices are DEPLOY, BUILD, and RUNTIME.
```
Comment (kcarmichael08) [inline generated/storage/policy.pb.go:409]: lifecycle is misspelled

File: generated/storage/policy.pb.go [hunk 20]
```diff
@@ -394,31 +394,44 @@ type Policy struct {
...
 	LifecycleStages []LifecycleStage `protobuf:"varint,9,rep,packed,name=lifecycle_stages,json=lifecycleStages,proto3,enum=storage.LifecycleStage" json:"lifecycle_stages,omitempty" search:"Lifecycle Stage,store"` // @gotags: search:"Lifecycle Stage,store"
-	EventSource     EventSource      `protobuf:"varint,22,opt,name=event_source,json=eventSource,proto3,enum=storage.EventSource" json:"event_source,omitempty"`
-	Exclusions      []*Exclusion     `protobuf:"bytes,21,rep,name=exclusions,proto3" json:"exclusions,omitempty"`
-	Scope           []*Scope         `protobuf:"bytes,11,rep,name=scope,proto3" json:"scope,omitempty"`
-	Severity        Severity         `protobuf:"varint,12,opt,name=severity,proto3,enum=storage.Severity" json:"severity,omitempty" search:"Severity,store"` // @gotags: search:"Severity,store"
+	// Describes which events should trigger execution of this policy
+	EventSource EventSource `protobuf:"varint,22,opt,name=event_source,json=eventSource,proto3,enum=storage.EventSource" json:"event_source,omitempty"`
+	// Define deployments or images that should be excluded from this policy.
```
Comment (kcarmichael08) [inline generated/storage/policy.pb.go:413]: define - should be "defines"?

File: generated/storage/policy.pb.go [hunk 21]
```diff
@@ -394,31 +394,44 @@ type Policy struct {
...
 	// FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates.
-	EnforcementActions []EnforcementAction    `protobuf:"varint,13,rep,packed,name=enforcement_actions,json=enforcementActions,proto3,enum=storage.EnforcementAction" json:"enforcement_actions,omitempty" search:"Enforcement"` // @gotags: search:"Enforcement"
-	Notifiers          []string               `protobuf:"bytes,14,rep,name=notifiers,proto3" json:"notifiers,omitempty"`
-	LastUpdated        *timestamppb.Timestamp `protobuf:"bytes,15,opt,name=last_updated,json=lastUpdated,proto3" json:"last_updated,omitempty" search:"Policy Last Updated"` // @gotags: search:"Policy Last Updated"
+	// Lists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT.
```
Comment (kcarmichael08) [inline generated/storage/policy.pb.go:422]: see previous comment on this string

File: generated/api/v1/policy_service.swagger.json [hunk 22]
```diff
@@ -649,64 +649,75 @@
...
+          },
+          "description": "List of categories that this policy falls under.  Category names must already exist in Central."
         },
         "lifecycleStages": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/storageLifecycleStage"
-          }
+          },
+          "description": "Describes which policy lifecylce stages this policy applies to.  Choices are DEPLOY, BUILD, and RUNTIME."
```
Comment (kcarmichael08) [inline generated/api/v1/policy_service.swagger.json:682]: lifecycle misspelled

File: generated/api/v1/policy_service.swagger.json [hunk 23]
```diff
@@ -975,64 +987,75 @@
...
```
Comment (kcarmichael08) [inline generated/api/v1/policy_service.swagger.json:1020]: lifecycle misspelled

File: image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml [hunk 24]
```diff
@@ -107,8 +123,12 @@ spec:
                   type: object
                 type: array
               isDefault:
+                description: Read-only field. Indicates the policy is a default policy
+                  if true and a custom policy if false.
                 type: boolean
               lifecycleStages:
+                description: LifecycleStages describes which policy lifecylce stages
```
Comment (kcarmichael08) [inline image/templates/helm/stackrox-central/crds/config.stackrox.io_securitypolicies.yaml:130]: lifecycle is misspelled - I'll stop marking, can do a search & replace for this

**Review by kcarmichael08 — COMMENTED** [review]
These are really clear and easy to understand - will be super helpful for users! I marked some things and then realized halfway through that they are the same strings, so I stopped marking. I guess you can do a search and replace for the ones that I've marked to get all of them. Just a few minor errors and some suggestions.

**Pull request merged**